sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationLiveTests.cs

```diff
@@ -486,44 +497,5 @@ private void AssertDocumentTypeDictionariesAreEquivalent(IReadOnlyDictionary<str
                 AssertDocumentTypesAreEqual(docType1, docType2);
             }
         }
-
-        private void ValidateDocumentModelDetails(DocumentModelDetails model, string description = null, IReadOnlyDictionary<string, string> tags = null)
```

Comment:
Helper test method not needed anymore since validation has been moved directly into the `BuildDocumentModel` test method.